test/entrypoint.sh

```diff
@@ -1,30 +1,37 @@
 #!/bin/bash
 
+set -e -u
+
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 # start rsyslog
-service rsyslog start &&
+service rsyslog start
+
+wait_tcp_port() {
+    local host="$1" port="$2"
+
+    # see http://tldp.org/LDP/abs/html/devref1.html for description of this syntax.
+    while ! exec 6<>/dev/tcp/$host/$port; do
+	echo "$(date) - still trying to connect to $host:$port"
+	sleep 1
```

Comment:
That `|| exit` is covered by  `set -e -u` that the patch adds at the start of the script to exit on any error or unset variable.